templates/kotlin/Project.cshtml

```diff
@@ -56,7 +56,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <None Update="@(Model.NuGetPackageId).targets" Pack="True" PackagePath="build\$(TargetFramework);buildTransitive\$(TargetFramework)" />
+    <None Include="@(Model.NuGetPackageId).targets" Pack="True" PackagePath="@@(AndroidXNuGetTargetFolders)" />
```

Comment:
Is the duplicated @@ intentional?

Reply:
Yep!  See https://github.com/xamarin/AndroidX/pull/404#discussion_r720381880.  :)

Reply:
Wow, I have no idea how I missed that. Well, the earlier review comment as well as the file ending signaling that this isn't stock MSBuild logic. 😅

Reply:
No worries! I'd rather be too careful than not careful enough!